postgres/tests/test_metadata.py

```diff
@@ -33,6 +33,7 @@ def stop_orphaned_threads():
     DBMAsyncJob.executor = ThreadPoolExecutor()
 
 
+@pytest.mark.integration
```

Comment:
This should not be necessary as all tests in this file are already marked as integration through the pytestmark global variable